components/tekton/pipelines/run-vse-sync-test/run-vse-sync-test.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: run-vse-sync-test
+spec:
+  params:
+    - description: Name of target cluster's kubeconfig secret
+      name: kubeconfig-secret
+      type: string
+    - description: Namespace of target cluster's kubeconfig secret
+      name: kubeconfig-namespace
+      type: string
+    - description: Name kubeconfig secret in pipline namespace
+      name: kubeconfig-secret-dest
+      type: string
+    - description: gChat URL
+      name: webhook_url
+      type: string
+      # yamllint disable-line rule:line-length
+      default: "https://chat.googleapis.com/v1/spaces/AAAAiCQuo38/messages?key=AIzaSyDdI0hCZtE6vySjMm-WEfRq3CPzqKqqsHI&token=QjLrTKvtNWmxFlkce7kVXJu6pzzlLtACBo8I4xem-AU"
+  tasks:
+    - name: copy-cluster-kubeconfig
+      params:
+        - name: SCRIPT
+          value: |-
+
+            if ! oc get secret $(params.kubeconfig-secret-dest); then
+
+              oc extract -n $(params.kubeconfig-namespace) \
+                secrets/$(params.kubeconfig-secret) \
+                --to /tmp --confirm=True
+
+              oc create secret generic $(params.kubeconfig-secret-dest) \
+                --from-file=kubeconfig=/tmp/kubeconfig \
+                --from-file=raw-kubeconfig=/tmp/raw-kubeconfig \
+                --type=Opaque
+
+            fi
+        - name: VERSION
+          value: '4.12'
```

Comment:
Discussed this, this is the openshift-client used on the cluster running Pipelines (Tekton), for us its 4.12 for now.  Something to keep in mind to put into the README in case futures require running the pipeline on 4.13+ Hub clusters.